### PR TITLE
OSD-21752: Add interface parameter for pcap-collector managed-script

### PR DESCRIPTION
**Pull request by Dee-6777**
### What type of PR is this?

_feature_

### What this PR does / Why we need it?
It'll help add interface as a parameter in pcap-collecter managed-scripts and help fetch traffic on that network interface

### Which Jira/Github issue(s) does this PR fix?

_Resolves #[OSD-21752](https://issues.redhat.com/browse/OSD-21752)_

### Pre-checks (if applicable)

- [X] Validated the changes in a cluster
- [X] Included documentation changes with PR

### CODE REVIEW

File: scripts/CEE/pcap-collector/README.md [hunk 1]
```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
 Parameters:
 - TIME: In seconds. Must be less than or equal to 900 (15 mins)
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: The
```
Comment (bmeng) [inline scripts/CEE/pcap-collector/README.md]: Incomplete?

File: scripts/CEE/pcap-collector/README.md [hunk 2]
```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
...
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: The 
 
 ```bash
 # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
 ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
 
+# Capture traffic for a particular interface in a node
+ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=vxlan_sys_4789
```
Comment (bmeng) [inline scripts/CEE/pcap-collector/README.md:21]: maybe update the example as `-p INTERFACE=all` as the vxlan is the default value anyway.

File: scripts/CEE/pcap-collector/README.md [hunk 3]
```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
...
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: It will add network interface as parameter
 
 ```bash
 # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
 ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
 
+# Capture traffic for a particular interface in a node
+ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=all
```
Comment (hectorakemp) [inline scripts/CEE/pcap-collector/README.md:21]: The way to specify all interfaces is `any` in tcpdump (see [link](https://serverfault.com/questions/805006/tcpdump-on-multiple-interfaces)). 

```suggestion
# Capture traffic for all interfaces on a node
ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal -p INTERFACE=any
```

**Review by hectorakemp — COMMENTED** [review]
Looks great! Just one small suggestion for the readme, then I think it's GTM. Awesome work @Dee-6777!!

File: scripts/CEE/pcap-collector/README.md [hunk 4]
```diff
@@ -10,15 +10,19 @@ __NOTE:__ backplane cli >= 0.0.23 is needed.
...
 - TIME: In seconds. Must be less than or equal to 900 (15 mins)
 - NODE: The node to run the packet capture on.
+- FILTERS: A valid set of PCAP filters for use in tcpdump
+- INTERFACE: It will add network interface as parameter
 
 ```bash
 # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
 ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
 
+# Capture traffic for a particular interface in a node
```
Comment (hectorakemp) [inline scripts/CEE/pcap-collector/README.md:20]: Should this be `Capture traffic for all interfaces on a node`?

**Review by hectorakemp — CHANGES_REQUESTED** [review]

**Review by hectorakemp — APPROVED** [review]
/lgtm
/approve

**Pull request merged**